src/mds/MDBalancer.cc

```diff
@@ -162,7 +162,10 @@ mds_load_t MDBalancer::get_load(utime_t now)
   ifstream cpu("/proc/loadavg");
   if (cpu.is_open())
     cpu >> load.cpu_load_avg;
-
+  else
+    derr << "input file '/proc/loadavg' not found" << dendl;
+  cpu.close();  
```

Comment:
I change it :)
the destructor will release it, do not need to display the call “cpu.close()”